appendix.ru.md

```diff
@@ -6,26 +6,26 @@ lang: ru
+Взгляды Ричарда М. Столлмана на законы об изнасиловании и педофилии были публично обсуждены осенью 2019 года, когда о них написала Селам Ж. [1] Особенно пугает то, как Столлман относится к обвинениям в том, что Марвин Минский изнасиловал одну из жертв торговли людьми Джеффри Эпштейна (Вирджиния Джюффре), сказав: «но наиболее вероятный сценарий событий состоит в том, что она представила себя полностью готовой к интимным связям»  [2]. Ричард М. Столлман заявляет, что это не «сексуальное насилие», потому что «нападение предполагает, что нападающий применил силу или принуждение», в то время как в обсуждаемом отчете «нет ничего подобного. Только то, что у них был секс.». [3] Вместо того, чтобы обсуждать это дальше, лучше сосредоточиться на его личном веб-сайте, где он делится своими взглядами на то, что несовершеннолетние «полностью готовы к интимным отношениям». [4]. (Примечание. Хотя несколько новостных сообщений и искажали позицию Ричарда М. Столлмана на тему обсуждения обвинений против Марвина, Ричард М. Столлман ранее сам высказывал мнение, которое полностью соответствовало искаженному.)
 
 [1]: https://web.archive.org/web/20210325013429/https://selamjie.medium.com/remove-richard-stallman-fec6ec210794
 [2]: https://web.archive.org/web/20210325013629/https://www.vice.com/en/article/9ke3ke/famed-computer-scientist-richard-stallman-described-epstein-victims-as-entirely-willing
 [3]: https://web.archive.org/web/20210325013629/https://www.vice.com/en/article/9ke3ke/famed-computer-scientist-richard-stallman-described-epstein-victims-as-entirely-willing
 [4]: https://web.archive.org/web/20210325013706/https://stallman.org/archives/2018-jul-oct.html#23_September_2018_(Cody_Wilson)
 
-Он неоднократно комментировал «нечестный закон, который, квалифицирует секс с подростками как "изнасилование", даже если они хотят этого». [5] Он сравнивает законы США с Суданскими законами, говоря, что «законы США, определяющие изнасилование, так же включают добровольный секс с детьми младше N лет (где N - варьируется)» и что «оба закона искажают значение слова изнасилование» [6].
+Он неоднократно комментировал «нечестный закон, который квалифицирует секс с подростками как 'изнасилование', даже если они хотят этого». [5] Он сравнивает законы США с Суданскими законами, говоря, что «законы США, определяющие изнасилование, так же включают добровольный секс с детьми младше N лет (где N - варьируется)» и что «оба закона искажают значение слова изнасилование» [6].
```

Comment:
„“

Reply:
```suggestion
Он неоднократно комментировал «нечестный закон, который квалифицирует секс с подростками как „изнасилование“, даже если они хотят этого». [5] Он сравнивает законы США с Суданскими законами, говоря, что «законы США, определяющие изнасилование, так же включают добровольный секс с детьми младше N лет (где N - варьируется)» и что «оба закона искажают значение слова изнасилование» [6].
```

Reply:
In original text `''`

Reply:
I doesn’t matter. In Russian, nested quotation marks are «„“».

Reply:
Okay.

Reply:
Just to note my voice: typographical quotes are not my strong suite, so on those points I defer to Andrew who seems to know them better.

Reply:
Fixed.